metpy/plots/_util.py

```diff
@@ -69,17 +71,76 @@ def add_logo(fig, x=10, y=25, zorder=100, size='small', **kwargs):
        The `figure` instance used for plotting
 
     """
-    fnames = {'small': 'metpy_75x75.png',
-              'large': 'metpy_150x150.png'}
+    fname_suffix = {'small': '_75x75.png',
+                    'large': '_150x150.png'}
+    fname_prefix = {'unidata': 'unidata',
+                    'metpy': 'metpy'}
     try:
-        metpy_logo = imread(os.path.join(os.path.dirname(__file__), '_static', fnames[size]))
+        fname = fname_prefix[which] + fname_suffix[size]
```

Comment:
Instead of using files/paths directly, need to use `pkg_resources`, like:

https://github.com/Unidata/MetPy/blob/73ac07bd32715f36bf68252cd54922eb4b237aeb/metpy/plots/ctables.py#L127-L143